app/Http/Requests/NameChangeRequest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class NameChangeRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|string|min:1|max:255',
```

Comment:
This is actually good to have max of 255 for name length. Could you add this to the other requests that take in a name? Or if you'd prefer to do that in a separate PR just make an issue for it to track it.

Reply:
I will do that

Reply:
#91

Reply:
#92  - pr for this issue